scripts/core/services/pageTitle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PageTitle service
+ *
+ * PageTitle service is used to save and update the title of the page.
+ * The title will be 'Superdesk {- Url} {-Desk/Workspace}
+ * Url can be: Dashboard, Monitoring, Spike, Highlights, Search, Settings
+ * If Url is search then there won't be desk
+ *
+ */
+export default angular.module('superdesk.core.services.pageTitle', [])
+    .service('pageTitle', function() {
+        this.title = 'Superdesk';
+        this.url = '';
+        this.workspace = '';
+
+        this.setUrl = function(url) {
```

Comment:
I would still just call this `setName` or something else, since it's not really a URL in the context of the entire application. If it were to be a URL, we wouldn't have to set it and we could make it work automatically by hooking it up to the router. What do you think? Amend as you decide and feel free to merge imo.